src/server/JasmineGraphInstanceService.cpp

```diff
@@ -80,4301 +144,5243 @@ void *instanceservicesession(void *dummyPt) {
+                    instance_logger.log("Error reading from socket", "error");
                 }
-            }
-            while (true) {
-                write(connFd, JasmineGraphInstanceProtocol::BATCH_UPLOAD_CHK.c_str(),
-                      JasmineGraphInstanceProtocol::BATCH_UPLOAD_CHK.size());
-                instance_logger.log("Sent : " + JasmineGraphInstanceProtocol::BATCH_UPLOAD_CHK, "info");
-                bzero(data, INSTANCE_DATA_LENGTH + 1);
-                read(connFd, data, INSTANCE_DATA_LENGTH);
-                line = (data);
 
-                if (line.compare(JasmineGraphInstanceProtocol::BATCH_UPLOAD_WAIT) == 0) {
-                    instance_logger.log("Received : " + JasmineGraphInstanceProtocol::BATCH_UPLOAD_WAIT, "info");
-                    sleep(1);
-                    continue;
-                } else if (line.compare(JasmineGraphInstanceProtocol::BATCH_UPLOAD_ACK) == 0) {
-                    instance_logger.log("Received : " + JasmineGraphInstanceProtocol::BATCH_UPLOAD_ACK, "info");
-                    instance_logger.log("Trained Model Batch upload completed", "info");
-                    break;
+                result_wr = write(sockfd, workerList.c_str(), workerList.size());
+
+                if (result_wr < 0) {
+                    instance_logger.log("Error writing to socket", "error");
                 }
-            }
-            loop = true;
-        } else if (line.compare(JasmineGraphInstanceProtocol::INITIATE_FRAGMENT_RESOLUTION) == 0) {
-            instance_logger.log("Received : " + JasmineGraphInstanceProtocol::INITIATE_FRAGMENT_RESOLUTION, "info");
-            write(connFd, JasmineGraphInstanceProtocol::OK.c_str(), JasmineGraphInstanceProtocol::OK.size());
-            instance_logger.log("Sent : " + JasmineGraphInstanceProtocol::OK, "info");
-            bzero(data, INSTANCE_DATA_LENGTH + 1);
-            read(connFd, data, INSTANCE_DATA_LENGTH);
-            string listOfPartitions = (data);
-            listOfPartitions = Utils::trim_copy(listOfPartitions, " \f\n\r\t\v");
-            instance_logger.log("Received ===>: " + listOfPartitions, "info");
-            std::stringstream ss;
-            ss << listOfPartitions;
-            while (true) {
-                write(connFd, JasmineGraphInstanceProtocol::FRAGMENT_RESOLUTION_CHK.c_str(),
-                      JasmineGraphInstanceProtocol::FRAGMENT_RESOLUTION_CHK.size());
-                instance_logger.log("Sent : " + JasmineGraphInstanceProtocol::FRAGMENT_RESOLUTION_CHK, "info");
 
-                bzero(data, INSTANCE_DATA_LENGTH + 1);
-                read(connFd, data, INSTANCE_DATA_LENGTH);
-                string listOfPartitions = (data);
+                instance_logger.log("Sent : Host List ", "info");
 
-                if (listOfPartitions.compare(JasmineGraphInstanceProtocol::FRAGMENT_RESOLUTION_DONE) == 0) {
-                    break;
+                bzero(data, 301);
+                read(sockfd, data, 300);
```

Comment:
```suggestion
                read(sockfd, data, INSTANCE_DATA_LENGTH);
```

Reply:
All such occurrences are replaced in next PR

Reply:
Ok